runtime/server/build.gradle.kts

```diff
@@ -44,6 +44,8 @@ dependencies {
   runtimeOnly("org.postgresql:postgresql")
   runtimeOnly(project(":polaris-relational-jdbc"))
   runtimeOnly("io.quarkus:quarkus-jdbc-postgresql")
+  implementation(libs.mysql.connector.j)
```

Comment:
We cannot ship this dependency it seems.

See this comment:

https://github.com/apache/polaris/issues/2491#issuecomment-3253157608

\cc @jbonofre

Reply:
Yes, the connector license is not compatible with ASF, AFAIK.

Reply:
Everything _except_ the driver is acceptable, I think.

So, @Jayden-Chiu : you may want to keep the java code changes, but remove the driver dependency. That would be mergeable from my POV (pending reviews from other maintainers, of course). Then, instructions on bundling MySQL Connector in a local build could be much simpler.